staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -826,6 +826,7 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 			if err != nil {
 				return nil, err
 			}
+			reqScope.FieldManager = fieldmanager.NewLazyFieldManager(reqScope.FieldManager, reqScope.Creater, reqScope.Kind)
```

Comment:
I like that we have a different set of dependencies for the lazy one than for the non-lazy one. That's good. Avoids having an increasing number of members in the regular one.

Reply:
I find it a little confusing that you put the fieldmanager into reqScope and then replace it right away, maybe use a temporary instead?